baseline/client/tests/spatial-orientation.test.js

```diff
@@ -1,7 +1,70 @@
 import { SpatialOrientation } from "../spatial-orientation/spatial-orientation.js";
+import { pressKey } from "./utils.js"
+import "jest-canvas-mock";
+
+const clickIcirc = (icirc, x, y) => {
+    const rect = icirc.getBoundingClientRect();
+    icirc.dispatchEvent(new MouseEvent("click", {
+        clientX: +x + icirc.width/2 + rect.left,
+        clientY: -y + icirc.height/2 + rect.top,
+    }));
+};
 
 describe("spatial-orientation", () => {
-    it.todo("results should have at least one result marked isRelevant");
+    it("results should have at least one result marked isRelevant", () => {
+        jest.useFakeTimers("legacy");
+        const timeline = (new SpatialOrientation(1)).getTimeline();
+        jsPsych.init({timeline: timeline});
+        timeline.forEach(() => {
+            pressKey(" ");
+            clickIcirc(document.getElementById("jspsych-spatial-orientation-icirc"), 0, 0);
+            jest.runAllTimers();
+        });
+        const progress = jsPsych.progress();
+        expect(progress.current_trial_global).toBe(progress.total_trials);
+        expect(jsPsych.data.get().filter({isRelevant: true}).values().length).toBeGreaterThan(0);
+    });
+
+    it("completes test trials with timedout and skipped if 5 minutes have elapsed", () => {
+        const advanceDateNow = (dateNow, ms) => () => dateNow() + ms;
+        const originalDateNow = Date.now();  // original Starwalker
```

Comment:
This is an actual timestamp. I think what you want is the function, which would just be Date.now (no parens).